WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageView.swift

```diff
@@ -53,6 +55,26 @@ struct AddProductFromImageView: View {
                     .lineLimit(5)
                     .fixedSize(horizontal: false, vertical: true)
             }
+
+            // Scanned text list.
+            Section {
+                VStack(alignment: .leading) {
+                    // Button to regenerate product details based on the selected scanned texts.
+                    Button(Localization.regenerateButtonTitle) {
+                        viewModel.generateProductDetails()
+                    }
+                    .buttonStyle(PrimaryLoadingButtonStyle(isLoading: viewModel.isGeneratingDetails))
+
+                    // Info text about selecting/editing the scanned text list.
+                    Text(Localization.scannedTextListInfo)
+                        .foregroundColor(.init(uiColor: .secondaryLabel))
+                        .captionStyle()
+                }
+                List(viewModel.scannedTexts) { scannedText in
```

Comment:
🔍  It's easy to accidentally toggle the check when tapping the rows to start editing (e.g. by tapping the space above the text field instead of directly on it). My guess is that list row behavior would prioritize button tap over text field. I tried replacing the `List` with `ScrollView` and it seems better. Please try [this diff](https://github.com/woocommerce/woocommerce-ios/compare/feat/10180-scanned-text-list...try/scanned-text-in-scrollview?expand=1) to see if it's better.

Reply:
Thanks for noticing the tap issue and the suggested diff! I noticed some issues using the diff:
- Using an overlay with an HStack means that long text in the text field can overlap with the toggle
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/37e34d2b-749f-4f0a-91ca-b5c376b19fa7" width="300" />
- Using a `List` comes with some handy UX like: when tapping on a row at the bottom it scrolls to the text field above the keyboard. I think it's still nice using a `List` unless there are some blocking issues

For the tap issue where tapping on the vertical space toggles the button: it looks like it's from using the default button style from [this similar SO post](https://stackoverflow.com/a/59402642/9185596). I updated the button to use `TextButtonStyle` in https://github.com/woocommerce/woocommerce-ios/pull/10220/commits/6b2d344bc19b2b099982bf1178a8f1ff2003f2e2 and it seems to be fixed 😄